pkg/virt-operator/kubevirt.go

```diff
@@ -78,10 +80,15 @@ func NewKubeVirtController(
 	operatorNamespace string,
 ) *KubeVirtController {
 
+	rl := workqueue.NewMaxOfRateLimiter(
+		workqueue.NewItemExponentialFailureRateLimiter(5*time.Second, 1000*time.Second),
+		&workqueue.BucketRateLimiter{Limiter: rate.NewLimiter(rate.Every(5*time.Second), 1)},
+	)
+
```

Comment:
nice. I didn't realize the queue had this functionality. I need to update some of my code now in other places.